toolkit/resources/manifests/package/pkggen_core_x86_64_1.0.txt

```diff
@@ -0,0 +1,166 @@
+filesystem-1.1-7.cm1.x86_64.rpm
+kernel-headers-5.10.21.1-3.cm1.noarch.rpm
+glibc-2.28-17.cm1.x86_64.rpm
```

Comment:
Is this introducing a new manifest?  Do we really need these?

Reply:
Yes and yes. New manifest allows us to build the 1.0 changes using the 1.0 packages. This prevents issues with a build when some package has been updated on a branch, but the package hasn't been built and released yet. It also makes sure that we use the same toolchain for all 1.0 builds of the same release.

Reply:
Won't this force us to remember about updating this set every time we update `1.0`? Could we instead get the `1.0` version during the build instead (only when we want to build against the `1.0` toolchain, of course)?